src/Angor/Avalonia/Angor.Contexts.CrossCutting/Angor.Contexts.CrossCutting.csproj

```diff
@@ -11,6 +11,7 @@
     <ItemGroup>
         <PackageReference Include="AspectInjector" />
         <PackageReference Include="CSharpFunctionalExtensions" />
+        <PackageReference Include="JetBrains.Annotations" />
```

Comment:
What is this added for?

Reply:
Ah, yes! This package reference was only being used for a single [UsedImplicitly] attribute. I agree it's unnecessary overhead for such limited usage.

I'll remove this reference completely. Thanks for pointing it out!